.eslintignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+!.github
```

Comment:
Dot files or folders are ignored by `eslint` by default. I think we should exclude this one as we have many node scripts in the `workflows/script` dir.